JS/ViewControllers/settingsViewController.js

```diff
@@ -0,0 +1,424 @@
+                              <!-- Switch -->
+                              <div class="switch">
+                                <label>
+                                  Forward
+                                  <input type="checkbox" name="direction" id="direction">
+                                  <span class="lever"></span>
+                                  Random
+                                </label>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                        <div class="row">
+                          <div class="col s4">
+                            <p>Script</p>
+                          </div>
+                          <div class="col s8">
+                            <p>
+                              <label>
+                                <input type="checkbox" class="filled-in" name="script" id="script-check"/>
+                                <span></span>
+                              </label>
+                            </p>
+                          </div>
+                        </div>
+                        <div class="row">
+                          <div class="col s4">
+                            <p>Translation</p>
+                          </div>
+                          <div class="col s8">
+                            <p>
+                              <label>
+                                <input type="checkbox" class="filled-in" name="translation" id="trans-check"/>
+                                <span></span>
+                              </label>
+                            </p>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+            <div class="row">
+              <div class="col s12">
+                <div class="divider"></div>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col s12 offset-l3 l6">
+                <div class="input-field col s12">
+                ` + langOptions + `
+                  <label>Paralel text</label>
+                </div>
+              </div>
+              <div class="col s12 center-align">
+                <a class="waves-effect waves-light btn-large" id="start-lecture"><i class="material-icons right">play_arrow</i>Play</a>
+              </div>
+            </div>
+        </div>
+        `;
+        super.renderHtml(htmlView);
+
+        var elems = document.querySelectorAll('select');
+        var instances = M.FormSelect.init(elems);
+
+        this.setupEventListeners();
+    }
+
+    setupProperties() {
+
+    }
+
+    setupEventListeners() {
+      document.getElementById('back-to-lecture-select').onclick = (e) => {
+        e.preventDefault();
+        this.navigationController.presentBefore();
```

Comment:
tato metoda je len setup, tu treba nabindovat ten vytiahnuty back-to-lecture-select element ktory uz mas odchyteny v setupProperties metode a priradit mu nejaku metodu ktora bude dole private